### ISSUE TITLE
Static naming style applying to const variables

### ISSUE THREAD

**Issue by 333fred** [dotnet/roslyn#15428]
I have the following naming spec set up:
![image](https://cloud.githubusercontent.com/assets/2371880/20500185/56554a24-afe9-11e6-8ec4-713a40082ff1.png)
This spec is being applied to a `s_camelCase` style.

I'm getting a naming style violation on `private const` identifiers like this one:
`private const string DebugPropertyClassId = "{0273C280-1882-4ED0-9308-52914672E3AA}"`. Running d15rel/25920.00.

**Comment by dpoeschl** [comment]
@333fred Yep, this is unfortunate. As a workaround, put a const field rule above the static field rule (even if it has severity "None", just to get them to not be included in the static field rule).

**Comment by dpoeschl** [comment]
@Pilchie Should this be pushed out of RTM given the existence of a fairly simple workaround?

**Comment by Pilchie** [comment]
How would a user discover that workaround?
